parserator_web/static/js/index.js

```diff
@@ -1,2 +1,53 @@
 /* TODO: Flesh this out to connect the form to the API and render results
    in the #address-results div. */
+
+var divblock = document.getElementById("address-results");
+var form = document.querySelector(".form");
+var t = document.getElementById('table-body');
+var parseType = document.getElementById('parse-type');
+
+// when the form is submitted, send API request
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  let apiurl = '/api/parse/' + '?address=' + document.getElementById("address").value;
+  fetch(apiurl)
+  .then((response) => response.json())
+  .then((data) => {
+    let rowhtml = "";
+    //if request is successful
+    if((data.Address)){
```

Comment:
Can you think of a more explicit check for success, perhaps a status code or another indication whether an error occurred?

Reply:
Yes, that makes much more sense! I added an if statement to check the status code of the response. a6f3271c0ac4189972819d9eda4570ba921fbd9e